versions.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "version": "0.3.0",
+    "title": "0.3.0",
+    "aliases": ["stable"]
+  },
+  {
+    "version": "dev",
+    "title": "dev",
+    "aliases": ["latest"]
+  }
+]
```

Comment:
Will this need to be manually updated as well, or does mike autogenerate it?

Reply:
This is created and updated by `mike`, but we can also manually edit it if necessary.